### ISSUE TITLE
Live stream to someone else's channel

### ISSUE THREAD

**Issue by rashikaranpuria** [youtube/yt-watchme#56]
I tried youtube's watchme app for live streaming and I understand the code fairly. In my use case the user needs to be able to live stream to another channel. I understand there is a need of Stream key here, but I need a rough guidance on where I need to change in the code. Any hints or a rough idea would do too. I just need headstart.

**Comment by sabodh** [comment]
Have you able to login.?

**Comment by rashikaranpuria** [comment]
@sabodh, yes I was able to login. My code is working now for streaming on your own account

**Comment by rashikaranpuria** [comment]
Yeah I was able to login but can't push content to someone's channel.

**Comment by sabodh** [comment]
@rashikaranpuria ,I am also trying to build the code but not able to login. Have you modified anything or added any thing ?

**Comment by rashikaranpuria** [comment]
@sabodh what error are you getting?

**Comment by sabodh** [comment]
@rashikaranpuria  Connections to the play services Failed. Could you please help me to fix this

**Comment by rashikaranpuria** [comment]
Yes, you need to sign the release apk.

**Comment by sabodh** [comment]
@rashikaranpuria  Ok, currently i am in debug mode I will check that.

**Comment by rashikaranpuria** [comment]
Apart from that I hope you have added client.json file in your repo.

**Comment by sabodh** [comment]
@rashikaranpuria  No I haven't added the file, whats the purpose of that file?

**Comment by rashikaranpuria** [comment]
See, that's the file you get if you want to enable Google login in your app.

**Comment by rashikaranpuria** [comment]
Check Google login tutorial for Android

**Comment by rashikaranpuria** [comment]
https://github.com/youtube/yt-watchme/issues/16#issuecomment-66397803

**Comment by sabodh** [comment]
@rashikaranpuria I will check . Thanks for the help

**Comment by sabodh** [comment]
Hi @rashikaranpuria , the application shows streaming .How can i verify the stream from youtube ?

**Comment by sabodh** [comment]
@rashikaranpuria  Its working Thanks for the support

**Comment by nomi2013** [comment]
From where i can get these folders and necessary files

libavcodec
ibavformat
libavutil
libavcodec

And lame also
Thanks

**Comment by Daria010291** [comment]
If you want to know how to set up your microphone for streaming on Youtube & Twitch, read this article: https://www.microphonetopgear.com/how-to-set-up-microphone-for-streaming/

**Comment by imran855** [comment]
Hi @rashikaranpuria , I need to stream someone else's channel.
Could you please share your findings? Is it possible or not?
